src/commands/Games/higherLower.ts

```diff
@@ -0,0 +1,246 @@
+				.setTitle(`Your won!`)
+				.setDescription(`You did it! The number was ${game.number}. Want to continue? ${this.calculateWinnings(game.wager, game.turn)}${Emojis.Shiny} are on the line`)
+				.setFooter('Act fast! You don\'t have much time')
+		});
+		console.log('filtering emotes');
+		const reactArray = Object.values(ReactionEmoji).filter(e => e !== ReactionEmoji.HIGHER && e !== ReactionEmoji.LOWER && e !== ReactionEmoji.CASHOUT);
+		for (const emoji of reactArray) {
+			await game.gameMessage.react(emoji);
+		}
+		console.log('GETTING EMOTES');
+		const reactionMap = await game.gameMessage.awaitReactions((r, u) => reactArray.includes(r.emoji.name) && u.id === user.id, { max: 1, time: 10000 });
+		await game.gameMessage.reactions.removeAll();
+
+		const whatToDo = reactionMap.size === 0
+			? endingAction.TIMEOUT
+			: reactionMap.first()!.emoji.name === ReactionEmoji.OK
+				? endingAction.PLAY
+				: endingAction.STOP;
+		switch (whatToDo) {
+			case endingAction.TIMEOUT:
+				await game.gameMessage.edit('Prompt timed out. Cashing out winnings...', { embed: null });
+			case endingAction.STOP:
+				await this.end(game, message, true);
+				break;
+			case endingAction.PLAY:
+				await game.gameMessage.edit('Alright. Starting new round', { embed: null });
+				break;
+		}
+		return whatToDo === endingAction.PLAY;
+	}
+
+	private async loss(game: HigherLowerGameData, message: KlasaMessage) {
+		// TODO(Quantum): Losing event
+		const { number, wager, turn, gameMessage } = game;
+		await gameMessage.reactions.removeAll();
+		let losings = wager;
+		if ((Math.random()) < 0.001) {
+			losings += this.calculateWinnings(wager, turn - 1);
+			await message.author.settings.decrease(UserSettings.Money, losings);
+		}
+		await gameMessage.edit(new MessageEmbed()
+			.setColor(getColor(message))
+			.setTitle(`You lost :\(`)
+			.setDescription(`You didn't quite get it. The number was ${number}. You lost ${losings}${Emojis.Shiny}`)
+			.setFooter('Better luck next time!'));
+		console.log('mesage should be edited');
+		return false;
+	}
+
+	private async end(game: HigherLowerGameData, message: KlasaMessage, cashout?: boolean) {
+		// Remove all the reactions and end the LLRC
+		await game.gameMessage.reactions.removeAll();
+		game.llrc.end();
+
+		// Should we need to cash out, proceed to doing that
+		if (cashout!) {
+			const { turn, wager } = game;
+			// Let the user know, and nullify the embed
+			await game.gameMessage.edit('Cashing out. Please hold...', { embed: null });
+
+			// Calculate and deposit winnings for that game
+			const winnings = this.calculateWinnings(wager, turn - 1);
+			await message.author.settings.increase(UserSettings.Money, winnings);
+
+			// Let the user know we're done!
+			await game.gameMessage.edit(`Paid out ${winnings}${Emojis.Shiny} to your account. Have fun!`);
+			return;
+		}
+
+		// Say bye!
+		await game.gameMessage.edit(new MessageEmbed()
+			.setColor('#DE0F1C')
```

Comment:
```suggestion
			.setColor('#C62828'')
```

Use the same red as Slotmachine please. Also while you're at it make it a static on SlotMachine as `SlotMachine.LOSE_COLOUR` and reference that here. And for the Green it should be `#00C853`